pkgs/development/tools/database/apgdiff/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchurl, makeWrapper, jre }:
```

```suggestion
{ lib
, stdenvNoCC
, fetchurl
, makeWrapper
, jre 
}:
```

- one arg per line makes future diffs easier to read
- if no CC is used, use `stdenvNoCC`